src/core_read.cpp

```diff
@@ -2,16 +2,13 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include <core_io.h>
-
```

Comment:
We always include the corresponding header file into the cpp file, because they are one module. For some reason core_io is the header file for both core_read and core_write...

Reply:
Oh, yes of course! Thanks for noticing. The unexpected file naming fooled me. Now fixed by adding a commit -- didn't squash in order to not invalidate the many utACK:s. Let me know if you prefer a squash anyway.